Mathlib/Topology/Order/Basic.lean

```diff
@@ -2573,6 +2573,93 @@ theorem exists_countable_dense_no_bot_top [SeparableSpace α] [Nontrivial α] :
 
 end DenselyOrdered
 
+section ConditionallyCompleteLinearOrder
+
+variable [ConditionallyCompleteLinearOrder α] [TopologicalSpace α] [OrderTopology α]
+  [ConditionallyCompleteLinearOrder β] [TopologicalSpace β] [OrderClosedTopology β] [Nonempty γ]
+
+/-- Sets are automatically bounded or cobounded in complete lattices. To use the same statements
+in complete and conditionally complete lattices but let automation fill automatically the
+boundedness proofs in complete lattices, we use the tactic `setIsBddDefault` in the statements,
+in the form `(hA : BddAbove A := by setIsBddDefault)`. -/
+
+macro "setIsBddDefault" : tactic =>
```

Comment:
I think it would be better to call it `bddDefault` (to match `BddAbove`/`BddBelow`) in namespace `Set` and move in just after [OrderBot.bddBelow](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Bounds/Basic.html#OrderBot.bddBelow)

Reply:
I could not get the namespace for the macro working by enclosing in a namespace or by directly preceding the name with `Set.`.

I moved the macro as suggested, and renamed it `setBddDefault` for now.

Reply:
Ah right, this doesn't work like in Lean3. I'd say just `bddDefault` should be fine.